src/modifiers/head_tail.py

```diff
@@ -1,13 +1,29 @@
-from talon import Context, Module
+from talon import Module
+from dataclasses import dataclass
 
 mod = Module()
-ctx = Context()
+mod.list("cursorless_head_tail", desc="Cursorless modifier for head or tail of line")
 
 
-mod.list("cursorless_head_tail", desc="Cursorless modifier for head or tail of line")
-ctx.lists["self.cursorless_head_tail"] = {"head", "tail"}
+@dataclass
+class HeadTail:
+    defaultSpokenForm: str
+    cursorlessIdentifier: str
+    type: str
+
+
+head_tail_list = [
+    HeadTail("head", "extendThroughStartOf", "head"),
+    HeadTail("tail", "extendThroughEndOf", "tail"),
```

Comment:
Hmm.  Yeah probably path of least resistance for now rather than having to mess around extension-side